### ISSUE TITLE
Sometimes kalabox tries to query non-existent containers.

### ISSUE THREAD

**Issue by stela5** [kalabox/kalabox#210]
ref: https://github.com/kalabox/kalabox/issues/167

Here's my verbose error output (Windows 7 Professional 64-bit):
```
kbox provision -- -v

...

debug: [15:18] Starting syncthing-image
debug: description => Installing your Syncthing image...
debug: dependencies => engine-up
info: -- Step 15/18 --
info: Installing your Syncthing image...
debug: EVENTS => Emitting event (pre-status)
debug: EVENTS => No listeners. pre-status
debug: B2D => Executing command. "C:\Program Files\Boot2Docker for Windows\boot2docker.exe" --hostip="10.13.37.1" status
debug: B2D => Command results. running

debug: EVENTS => Emitting event (post-status)
debug: EVENTS => No listeners. post-status
debug: EVENTS => Emitting event (pre-ip)
debug: EVENTS => No listeners. pre-ip
debug: B2D => Executing command. "C:\Program Files\Boot2Docker for Windows\boot2docker.exe" --hostip="10.13.37.1" ip
debug: B2D => Command results. 10.13.37.42

debug: EVENTS => Emitting event (post-ip)
debug: EVENTS => No listeners. post-ip
debug: DOCKER => initializing.  protocol=http, host=10.13.37.42, port=2375
info: DOCKER => Pulling image. name=kalabox/syncthing:stable, build=false, createOpts=undefined, startOpts=undefined, postProviderOpts=undefined
info: -- FAIL. --

connect ETIMEDOUT
Error: connect ETIMEDOUT
    at exports._errnoException (util.js:746:11)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1010:19)
```

```
C:\Windows\System32>boot2docker --vm="Kalabox2" ip
10.13.37.42
```

I'll try https://github.com/kalabox/kalabox/issues/167#issuecomment-111780618

**Comment by stela5** [comment]
Here's the result of my second attempt:
```
kbox down
kbox up
mkdir c:\tmp\kalabox\apps
cd c:\tmp\kalabox\apps
kbox create drupal
cd *APP_NAME*
kbox install
```
**The dependency "engineConfig" was NOT found!**

**Comment by stela5** [comment]
3rd attempt (after rebooting the computer):
```
kbox down
kbox up
cd c:\tmp\kalabox\apps\APP_NAME
kbox install
```

**The dependency "engineConfig" was NOT found!**

I'm new with kalabox so I'm not sure what exactly is going on or if the solution is simple.  Any help would be appreciated.

**Comment by stela5** [comment]
I tried `kbox provision -- -v` again without success (same error).

P.S. I'm on WiFi, if that makes a difference.

**Comment by stela5** [comment]
I tried `kbox provision -- -v` directly plugged into the router and also directly plugged into the modem and all result in the same error:

```
info: -- Step 15/18 --
info: Installing your Syncthing image...
debug: EVENTS => Emitting event (pre-status)
debug: EVENTS => No listeners. pre-status
debug: B2D => Executing command. "C:\Program Files\Boot2Docker for Windows\boot2docker.exe" --hostip="10.13.37.1" status
debug: B2D => Command results. running

debug: EVENTS => Emitting event (post-status)
debug: EVENTS => No listeners. post-status
debug: EVENTS => Emitting event (pre-ip)
debug: EVENTS => No listeners. pre-ip
debug: B2D => Executing command. "C:\Program Files\Boot2Docker for Windows\boot2docker.exe" --hostip="10.13.37.1" ip
debug: B2D => Command results. 10.13.37.42

debug: EVENTS => Emitting event (post-ip)
debug: EVENTS => No listeners. post-ip
debug: DOCKER => initializing.  protocol=http, host=10.13.37.42, port=2375
info: DOCKER => Pulling image. name=kalabox/syncthing:stable, build=false, createOpts=undefined, startOpts=undefined, postProviderOpts=undefined
info: -- FAIL. --

connect ETIMEDOUT
Error: connect ETIMEDOUT
    at exports._errnoException (util.js:746:11)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1010:19)
```

**Comment by stela5** [comment]
I also rebooted and tried again when connected directly into the modem without success.  At this point, I'll wait for the experts to weigh in...

**Comment by pirog** [comment]
i've seen different variants of this which are generally caused by not being able to connect to the docker registry which has all our images and containers. 

the second `engine not found` error is a result of not having provisioned with success. 

what i would do is wait on this for a few days. we are rolling a new release pretty soon that may address this issue.

**Comment by donkoller** [comment]
I noticed on Windows 8.1 that I was not successful with provisioning until I was logged in as an admin user. 
(Still cannot see the website in a browser, though.)

**Comment by pirog** [comment]
Yeah im not sure if there is an easy way to get around the admin user thing on windows. We have to run some commands in "elevated" mode in order to get the DNS to function but you can't run things in elevate mode with admin permission. 

Might be a way to prompt for admin credentials at some point. We've got a 8.1 box on the way so we can do some more testing there.

**Comment by pirog** [comment]
Hey @stela5, 

We just released 0.8.0 which may resolve this issue. I would try uninstalling (https://github.com/kalabox/kalabox/wiki/Uninstalling-Kalabox) pulling in the new code and provisioning using 

```
npm install kalabox -g
kbox provision
```

If you did the developer install you should consult https://github.com/kalabox/kalabox/wiki/Updating-Kalabox

**Comment by stela5** [comment]
Sorry, no luck:

```

C:\Users\me\AppData\Roaming\npm\node_modules\kalabox\node_modules\npm\lib\utils\correct-mkdir.js:37
    var uid = +process.getuid()
                       ^
TypeError: undefined is not a function
    at calculateOwner (C:\Users\me\AppData\Roaming\npm\node_modules\kalabox\node_modules\npm\lib\utils\correct-mkdir.js:37:24)
    at C:\Users\me\AppData\Roaming\npm\node_modules\kalabox\node_modules\npm\lib\utils\correct-mkdir.js:23:22
    at FSReqWrap.oncomplete (evalmachine.<anonymous>:95:15)
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "kalabox" "-g"
npm ERR! node v0.12.5
npm ERR! npm  v2.11.2
npm ERR! code ELIFECYCLE

npm ERR! kalabox@0.8.1 postinstall: `node ./scripts/postinstall.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the kalabox@0.8.1 postinstall script 'node ./scripts/postinstall.js'.
npm ERR! This is most likely a problem with the kalabox package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./scripts/postinstall.js
npm ERR! You can get their info via:
npm ERR!     npm owner ls kalabox
npm ERR! There is likely additional logging output above.
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "kalabox" "-g"
npm ERR! node v0.12.5
npm ERR! npm  v2.11.2
npm ERR! path C:\Windows\System32\npm-debug.log.c2a9b7e71233ef19b28bccb0c49722f0

npm ERR! code EPERM
npm ERR! errno -4048

npm ERR! Error: EPERM, open 'C:\Windows\System32\npm-debug.log.c2a9b7e71233ef19b28bccb0c49722f0'
npm ERR!     at Error (native)
npm ERR!  { [Error: EPERM, open 'C:\Windows\System32\npm-debug.log.c2a9b7e71233ef19b28bccb0c49722f0']
npm ERR!   errno: -4048,
npm ERR!   code: 'EPERM',
npm ERR!   path: 'C:\\Windows\\System32\\npm-debug.log.c2a9b7e71233ef19b28bccb0c49722f0' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Windows\System32\npm-debug.log
```

**Comment by pirog** [comment]
looks like a completely different error this time. going to open up a new issue for this so we can keep tracking the timeout issue.

**Comment by pirog** [comment]
@bcauldwell so sometimes we get this timeout error when we pull from the docker registry. Can you propose a resolution on how to handle this? Sometimes just restarting the pull resolves the issue. Sometimes it seems to require a kbox down and up to fix.

**Comment by pirog** [comment]
Not completely sure yet but i think this will be resolved after #216 is solved

**Comment by kalacommitbot** [comment]
@stela5 @donkoller i would try out the new 0.10.x release which makes some big changes on how these things are handled and may resolve the issue.

**Comment by stela5** [comment]
Sorry, no luck.  In addition to the unexpected version on the welcome screen (*Welcome to Kalabox 2!*), here's the error:

```

info: -- Step 23/23 --
info: Finishing install
info: -- OK! 100% complete! --

info: Huzzah! Victory!
info: BOOT2DOCKER =>  Starting up. maxRetries=3, max=3
info: BOOT2DOCKER =>  Initializing boot2docker [1].
info: BOOT2DOCKER =>  Bringing boot2docker up [1].
info: BOOT2DOCKER =>  Boot2docker is up.
Kalabox engine has been activated.
Error while installing app: "modern-form-client-demo": Error querying isRunning: "'138ecd2272b736adcb94043146f6561e76312b3628e579efd640c055ba75d176'".
: Object function lodash(value) {
      if (isObjectLike(value) && !isArray(value) && !(value instanceof LazyWrapper)) {
        if (value instanceof LodashWrapper) {
          return value;
        }
        if (hasOwnProperty.call(value, '__chain__') && hasOwnProperty.call(value, '__wrapped__')) {
          return wrapperClone(value);
        }
      }
      return new LodashWrapper(value);
    } has no method 'get'
VError: Error querying isRunning: "'138ecd2272b736adcb94043146f6561e76312b3628e579efd640c055ba75d176'".: Object function lodash(value) {
      if (isObjectLike(value) && !isArray(value) && !(value instanceof LazyWrapper)) {
        if (value instanceof LodashWrapper) {
          return value;
        }
        if (hasOwnProperty.call(value, '__chain__') && hasOwnProperty.call(value, '__wrapped__')) {
          return wrapperClone(value);
        }
      }
      return new LodashWrapper(value);
    } has no method 'get'
    at c:\tmp\node_modules\kalabox-engine-docker\docker.js.jx:300:13
From previous event:
    at Object.isRunning (c:\tmp\node_modules\kalabox-engine-docker\docker.js.jx:299:11)
From previous event:
    at c:\tmp\lib\engine.js.jx:345:29
From previous event:
    at Object.exports.isRunning.exports.isContainerRunning (c:\tmp\lib\engine.js.jx:344:4)
    at c:\tmp\node_modules\kalabox-services-kalabox\kalabox.js.jx:67:26
From previous event:
    at isServiceRunning (c:\tmp\node_modules\kalabox-services-kalabox\kalabox.js.jx:66:6)
    at serviceInfo.then.filter.concurrency (c:\tmp\node_modules\kalabox-services-kalabox\kalabox.js.jx:386:14)
From previous event:
    at Object.verify (c:\tmp\node_modules\kalabox-services-kalabox\kalabox.js.jx:385:6)
From previous event:
    at Object.exports.verify (c:\tmp\lib\services.js.jx:89:29)
    at Object.exports.install (c:\tmp\lib\app.js.jx:388:19)
    at Object.<anonymous> (c:\tmp\lib\create.js.jx:372:26)
From previous event:
    at c:\tmp\lib\create.js.jx:371:8
    at process._tickCallback (node.js:812:13)
From previous event:
    at createApp (c:\tmp\lib\create.js.jx:349:6)
    at PromptUI.completed (c:\tmp\lib\create.js.jx:475:18)
    at PromptUI.onCompletion (c:\tmp\node_modules\inquirer\lib\ui\prompt.js.jx:68:10)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.completed (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:1792:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:1729:1
4)
    at AnonymousObserver.tryCatcher (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:1729:
14)
    at InnerObserver.onCompleted (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:3424:65)
    at InnerObserver.tryCatcher (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:1729:
14)
    at AutoDetachObserver.tryCatcher (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:1729:
14)
    at InnerObserver.onCompleted (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:3424:65)
    at InnerObserver.tryCatcher (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\tmp\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:1729:
14)
```

**Comment by pirog** [comment]
I would try running `kbox create pantheon` again and see what happens. Use a different app name.

**Comment by pirog** [comment]
Changing the title here. I think we've resolved the original timeout issue but getting a new one. 

My guess on this one is we might be trying to query old CID files before they are rewritten with the correct CIDs, specifically regarding the startup services.

**Comment by stela5** [comment]
No luck:

```
? Which site? modern-form-client-demo
? Which environment? dev
? What will you call this monster you have created: demo-2
Building your app...
info: BOOT2DOCKER =>  Starting up. maxRetries=3, max=3
info: BOOT2DOCKER =>  Initializing boot2docker [1].
info: BOOT2DOCKER =>  Bringing boot2docker up [1].
info: BOOT2DOCKER =>  Boot2docker is up.
Kalabox engine has been activated.
Error while installing app: "demo-2": Error querying isRunning: "'138ecd2272b736adcb94043146f6561e76312b3628e579efd640c055ba75d176'".: Object function
 lodash(value) {
      if (isObjectLike(value) && !isArray(value) && !(value instanceof LazyWrapper)) {
        if (value instanceof LodashWrapper) {
          return value;
        }
        if (hasOwnProperty.call(value, '__chain__') && hasOwnProperty.call(value, '__wrapped__')) {
          return wrapperClone(value);
        }
      }
      return new LodashWrapper(value);
    } has no method 'get'
VError: Error querying isRunning: "'138ecd2272b736adcb94043146f6561e76312b3628e579efd640c055ba75d176'".: Object function lodash(value) {
      if (isObjectLike(value) && !isArray(value) && !(value instanceof LazyWrapper)) {
        if (value instanceof LodashWrapper) {
          return value;
        }
        if (hasOwnProperty.call(value, '__chain__') && hasOwnProperty.call(value, '__wrapped__')) {
          return wrapperClone(value);
        }
      }
      return new LodashWrapper(value);
    } has no method 'get'
    at c:\playground\node_modules\kalabox-engine-docker\docker.js.jx:300:13
From previous event:
    at Object.isRunning (c:\playground\node_modules\kalabox-engine-docker\docker.js.jx:299:11)
From previous event:
    at c:\playground\lib\engine.js.jx:345:29
From previous event:
    at Object.exports.isRunning.exports.isContainerRunning (c:\playground\lib\engine.js.jx:344:4)
    at c:\playground\node_modules\kalabox-services-kalabox\kalabox.js.jx:67:26
From previous event:
    at isServiceRunning (c:\playground\node_modules\kalabox-services-kalabox\kalabox.js.jx:66:6)
    at serviceInfo.then.filter.concurrency (c:\playground\node_modules\kalabox-services-kalabox\kalabox.js.jx:386:14)
From previous event:
    at Object.verify (c:\playground\node_modules\kalabox-services-kalabox\kalabox.js.jx:385:6)
From previous event:
    at Object.exports.verify (c:\playground\lib\services.js.jx:89:29)
    at Object.exports.install (c:\playground\lib\app.js.jx:388:19)
    at Object.<anonymous> (c:\playground\lib\create.js.jx:372:26)
From previous event:
    at c:\playground\lib\create.js.jx:371:8
    at process._tickCallback (node.js:812:13)
From previous event:
    at createApp (c:\playground\lib\create.js.jx:349:6)
    at PromptUI.completed (c:\playground\lib\create.js.jx:475:18)
    at PromptUI.onCompletion (c:\playground\node_modules\inquirer\lib\ui\prompt.js.jx:68:10)
    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.completed (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:1792:12)
    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx
:1729:14)
    at AnonymousObserver.tryCatcher (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.j
x:1729:14)
    at InnerObserver.onCompleted (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:3424:65)
    at InnerObserver.tryCatcher (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.j
x:1729:14)
    at AutoDetachObserver.tryCatcher (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.j
x:1729:14)
    at InnerObserver.onCompleted (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:3424:65)
    at InnerObserver.tryCatcher (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:566:29)
    at AutoDetachObserverPrototype.completed (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.jx:5287:56)
    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (c:\playground\node_modules\inquirer\node_modules\rx\dist\rx.js.j
x:1729:14)
```

**Comment by pirog** [comment]
:( 

Will look into this further and try to get a fix in for the 0.10.3 release.

**Comment by bcauldwell** [comment]
[issue.txt | uploaded via ZenHub](https://files.zenhub.io/560990854762504a68d1096b)

**Comment by bcauldwell** [comment]
[
  {
    "_id": "38ebeb35-3fbd-4cd4-85c9-da3628f10bfe",
    "created": "2015-09-26T00:56:22.195Z",
    "metaData": [
      {
        "created": "2015-09-26T07:53:48.755Z",
        "recorded": "2015-09-26T00:56:22.396Z",
        "data": {
          "action": "error",
          "message": "Error while installing app: \"stuff\": Error querying isRunning: \"false\".: Invalid cid: false",
          "stack": "VError: Error querying isRunning: \"false\".: Invalid cid: false\n    at C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\kalabox-engine-docker\\docker.js.jx:300:13\nFrom previous event:\n    at Object.isRunning (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\kalabox-engine-docker\\docker.js.jx:299:11)\nFrom previous event:\n    at C:\\Users\\pirog\\AppData\\Roaming\\npm\\lib\\engine.js.jx:345:29\nFrom previous event:\n    at Object.exports.isRunning.exports.isContainerRunning (C:\\Users\\pirog\\AppData\\Roaming\\npm\\lib\\engine.js.jx:344:4)\n    at C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\kalabox-services-kalabox\\kalabox.js.jx:67:26\nFrom previous event:\n    at isServiceRunning (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\kalabox-services-kalabox\\kalabox.js.jx:66:6)\n    at serviceInfo.then.filter.concurrency (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\kalabox-services-kalabox\\kalabox.js.jx:386:14)\nFrom previous event:\n    at Object.verify (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\kalabox-services-kalabox\\kalabox.js.jx:385:6)\nFrom previous event:\n    at Object.exports.verify (C:\\Users\\pirog\\AppData\\Roaming\\npm\\lib\\services.js.jx:89:29)\n    at Object.exports.install (C:\\Users\\pirog\\AppData\\Roaming\\npm\\lib\\app.js.jx:388:19)\n    at Object.<anonymous> (C:\\Users\\pirog\\AppData\\Roaming\\npm\\lib\\create.js.jx:403:26)\nFrom previous event:\n    at C:\\Users\\pirog\\AppData\\Roaming\\npm\\lib\\create.js.jx:402:8\n    at process._tickCallback (node.js:812:13)\nFrom previous event:\n    at createApp (C:\\Users\\pirog\\AppData\\Roaming\\npm\\lib\\create.js.jx:380:6)\n    at PromptUI.completed (C:\\Users\\pirog\\AppData\\Roaming\\npm\\lib\\create.js.jx:512:18)\n    at PromptUI.onCompletion (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\ui\\prompt.js.jx:68:10)\n    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.completed (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:1792:12)\n    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:1729:14)\n    at AnonymousObserver.tryCatcher (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:566:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:5287:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:1729:14)\n    at InnerObserver.onCompleted (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:3424:65)\n    at InnerObserver.tryCatcher (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:566:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:5287:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:1729:14)\n    at AutoDetachObserver.tryCatcher (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:566:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:5287:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:1729:14)\n    at InnerObserver.onCompleted (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:3424:65)\n    at InnerObserver.tryCatcher (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:566:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:5287:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:1729:14)\n    at AutoDetachObserver.tryCatcher (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:566:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:5287:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js.jx:1729:14)\n    at C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\utils\\utils.js.jx:52:13\n    at C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\lodash\\index.js.jx:7172:25\n    at C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\ui\\prompt.js.jx:97:7\n    at Object.utils.runAsync (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\utils\\utils.js.jx:36:5)\n    at Prompt.filter (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\prompts\\base.js.jx:183:9)\n    at Prompt.done (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\prompts\\base.js.jx:71:10)\n    at Prompt.<anonymous> (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\prompts\\input.js.jx:89:10)\n    at Object.utils.runAsync (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\utils\\utils.js.jx:36:5)\n    at Prompt.filter (C:\\Users\\pirog\\AppData\\Roaming\\npm\\node_modules\\inquirer\\lib\\prompts\\base.js.jx:183:9)",
          "os": {
            "type": "Windows_NT",
            "platform": "win32",
            "release": "6.1.7601",
            "arch": "x64"
          },
          "version": "0.10.2",
          "nodeVersion": "v0.10.40"
        }
      }
    ]
  },
  {
    "_id": "b7d568dd-8373-474a-a016-37c0071b76cf",
    "created": "2015-09-26T03:45:56.676Z",
    "metaData": [
      {
        "created": "2015-09-26T10:43:22.015Z",
        "recorded": "2015-09-26T03:45:56.860Z",
        "data": {
          "action": "error",
          "message": "App already exists: C:\\Users\\pirog\\Desktop\\apps\\playbox",
          "stack": "Error: App already exists: C:\\Users\\pirog\\Desktop\\apps\\playbox\n    at C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:341:17\nFrom previous event:\n    at C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:340:10\n    at processImmediate [as _immediateCallback] (timers.js:358:17)\nFrom previous event:\n    at C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:336:8\nFrom previous event:\n    at createApp (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:330:23)\n    at PromptUI.completed (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:512:18)\n    at PromptUI.onCompletion (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\ui\\prompt.js:69:10)\n    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1793:12)\n    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AnonymousObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at InnerObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:3425:65)\n    at InnerObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AutoDetachObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at InnerObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:3425:65)\n    at InnerObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AutoDetachObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:53:13\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\lodash\\index.js:7173:25\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\ui\\prompt.js:98:7\n    at Object.utils.runAsync (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:37:5)\n    at Prompt.filter (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:184:9)\n    at Prompt.done (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:72:10)\n    at Prompt.<anonymous> (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\input.js:90:10)\n    at Object.utils.runAsync (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:37:5)\n    at Prompt.filter (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:184:9)\n    at Prompt.onEnd (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\input.js:81:8)",
          "os": {
            "type": "Windows_NT",
            "platform": "win32",
            "release": "6.1.7601",
            "arch": "x64"
          },
          "version": "0.10.2",
          "nodeVersion": "v0.12.2"
        }
      },
      {
        "created": "2015-09-26T10:44:15.684Z",
        "recorded": "2015-09-26T03:46:50.548Z",
        "data": {
          "action": "error",
          "message": "Error while installing app: \"weg\": Error querying isRunning: \"'bc4036d624e876be0e9405b045f28db514e99791f32aaf5cb2da305a977bcde9'\".: Error inspecting container.: Error querying docker for list of containers.: connect ETIMEDOUT",
          "stack": "VError: Error querying isRunning: \"'bc4036d624e876be0e9405b045f28db514e99791f32aaf5cb2da305a977bcde9'\".: Error inspecting container.: Error querying docker for list of containers.: connect ETIMEDOUT\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-engine-docker\\docker.js:301:13\nFrom previous event:\n    at Object.isRunning (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-engine-docker\\docker.js:300:11)\nFrom previous event:\n    at C:\\Users\\pirog\\Desktop\\kalabox\\lib\\engine.js:345:29\nFrom previous event:\n    at Object.exports.isRunning.exports.isContainerRunning (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\engine.js:344:4)\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-services-kalabox\\kalabox.js:68:26\nFrom previous event:\n    at isServiceRunning (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-services-kalabox\\kalabox.js:67:6)\n    at serviceInfo.then.filter.concurrency (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-services-kalabox\\kalabox.js:387:14)\nFrom previous event:\n    at Object.verify (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-services-kalabox\\kalabox.js:386:6)\nFrom previous event:\n    at Object.exports.verify (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\services.js:89:29)\n    at Object.exports.install (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\app.js:388:19)\n    at Object.<anonymous> (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:403:26)\nFrom previous event:\n    at C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:402:8\n    at processImmediate [as _immediateCallback] (timers.js:358:17)\nFrom previous event:\n    at createApp (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:380:6)\n    at PromptUI.completed (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:512:18)\n    at PromptUI.onCompletion (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\ui\\prompt.js:69:10)\n    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1793:12)\n    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AnonymousObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at InnerObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:3425:65)\n    at InnerObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AutoDetachObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at InnerObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:3425:65)\n    at InnerObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AutoDetachObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:53:13\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\lodash\\index.js:7173:25\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\ui\\prompt.js:98:7\n    at Object.utils.runAsync (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:37:5)\n    at Prompt.filter (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:184:9)\n    at Prompt.done (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:72:10)\n    at Prompt.<anonymous> (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\input.js:90:10)\n    at Object.utils.runAsync (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:37:5)\n    at Prompt.filter (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:184:9)",
          "os": {
            "type": "Windows_NT",
            "platform": "win32",
            "release": "6.1.7601",
            "arch": "x64"
          },
          "version": "0.10.2",
          "nodeVersion": "v0.12.2"
        }
      },
      {
        "created": "2015-09-26T10:53:34.479Z",
        "recorded": "2015-09-26T03:56:09.293Z",
        "data": {
          "action": "error",
          "message": "Error while installing app: \"rge\": Error querying isRunning: \"'bc4036d624e876be0e9405b045f28db514e99791f32aaf5cb2da305a977bcde9'\".: Error inspecting container.: Error querying docker for list of containers.: connect ETIMEDOUT",
          "stack": "VError: Error querying isRunning: \"'bc4036d624e876be0e9405b045f28db514e99791f32aaf5cb2da305a977bcde9'\".: Error inspecting container.: Error querying docker for list of containers.: connect ETIMEDOUT\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-engine-docker\\docker.js:301:13\nFrom previous event:\n    at Object.isRunning (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-engine-docker\\docker.js:300:11)\nFrom previous event:\n    at C:\\Users\\pirog\\Desktop\\kalabox\\lib\\engine.js:345:29\nFrom previous event:\n    at Object.exports.isRunning.exports.isContainerRunning (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\engine.js:344:4)\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-services-kalabox\\kalabox.js:68:26\nFrom previous event:\n    at isServiceRunning (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-services-kalabox\\kalabox.js:67:6)\n    at serviceInfo.then.filter.concurrency (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-services-kalabox\\kalabox.js:387:14)\nFrom previous event:\n    at Object.verify (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\kalabox-services-kalabox\\kalabox.js:386:6)\nFrom previous event:\n    at Object.exports.verify (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\services.js:89:29)\n    at Object.exports.install (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\app.js:388:19)\n    at Object.<anonymous> (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:403:26)\nFrom previous event:\n    at C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:402:8\n    at processImmediate [as _immediateCallback] (timers.js:358:17)\nFrom previous event:\n    at createApp (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:380:6)\n    at PromptUI.completed (C:\\Users\\pirog\\Desktop\\kalabox\\lib\\create.js:512:18)\n    at PromptUI.onCompletion (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\ui\\prompt.js:69:10)\n    at AnonymousObserver.Rx.AnonymousObserver.AnonymousObserver.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1793:12)\n    at AnonymousObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AnonymousObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at InnerObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:3425:65)\n    at InnerObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AutoDetachObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at InnerObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:3425:65)\n    at InnerObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at AutoDetachObserver.tryCatcher (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:567:29)\n    at AutoDetachObserverPrototype.completed (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:5288:56)\n    at AutoDetachObserver.Rx.internals.AbstractObserver.AbstractObserver.onCompleted (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\node_modules\\rx\\dist\\rx.js:1730:14)\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:53:13\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\lodash\\index.js:7173:25\n    at C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\ui\\prompt.js:98:7\n    at Object.utils.runAsync (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:37:5)\n    at Prompt.filter (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:184:9)\n    at Prompt.done (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:72:10)\n    at Prompt.<anonymous> (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\input.js:90:10)\n    at Object.utils.runAsync (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\utils\\utils.js:37:5)\n    at Prompt.filter (C:\\Users\\pirog\\Desktop\\kalabox\\node_modules\\inquirer\\lib\\prompts\\base.js:184:9)",
          "os": {
            "type": "Windows_NT",
            "platform": "win32",
            "release": "6.1.7601",
            "arch": "x64"
          },
          "version": "0.10.2",
          "nodeVersion": "v0.12.2"
        }
      }
    ]
  }
]

**Comment by pirog** [comment]
Still can't replicate this so going to move into the backlog and try to resolve in alpha5

**Comment by pirog** [comment]
Still cant replicate. Going to close and hope we get more through error reporting.
